developers/addon/index.md

```diff
@@ -196,7 +206,7 @@ If a thing will be created for this thing type then its properties will be autom
 Therefore the `org.openhab.core.thing.Thing` interface also provides the `getProperties` operation to retrieve the defined properties.
 In contrast to the `getProperties` operation of the thing type instance the result of the thing´s `getProperties` operation will also contain the properties updated during runtime (cp. the thing handler [documentation](thing-handler.html)).
```

Comment:
The link will result in a 404.
Do you simply want to jump to the corresponding doc part of this page?

```suggestion
In contrast to the `getProperties` operation of the thing type instance the result of the thing´s `getProperties` operation will also contain the properties updated during runtime (cp. the thing handler [documentation](#the-thinghandler)).
```

Reply:
I have no idea where this should point to. It's already a 404 in the current version. But probably your suggestion is fine.